nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp create reaction

Description:
Backend for creating new reactions is complete

Testing Instructions:

Postman URL:
```
http://localhost:8000/reactions
```

Postman Object:
```
{
    "label": "wow",
    "imageUrl": "www.wow.com"
}
```

Make a POST request and see if the new object was added to the database.
